src/dblib_stubs.c

```diff
@@ -50,6 +50,30 @@ static void raise_fatal(char *msg)
   CAMLreturn0;
 }
 
+static int msg_handler(DBPROCESS *dbproc, DBINT msgno, int msgstate, int severity,
+		       char *msgtext, char *srvname, char *procname, int line)
```

Comment:
The handler is given more information than the message, e.g. the `severity` and the `line`.  Any reason not to pass them to the OCaml handler?